package.json

```
@@ -8,8 +8,8 @@
     "url": "https://github.com/nasa-jpl/explorer-1"
   },
   "engines": {
-    "node": "14.19.0",
-    "npm": "^6.14.16"
+    "node": "^16.20.2",
```

```suggestion
    "node": "^14.19.0",
```
This could be more forgiving and make gradual node updates possible. Otherwise we'll need to update all of our projects that use explorer-1 to npm 16.20.2